File: app/src/main/java/com/danilomendes/figurines/data/remote/api/CompanyService.java
package com.danilomendes.figurines.data.remote.api;

import com.danilomendes.figurines.data.entity.Company;

import java.util.List;

import io.reactivex.Single;
import retrofit2.http.GET;

/**
 * Created by danilo on 11-10-2017.
 */
public interface CompanyService {

    @GET("companies.json")
    Single<List<Company>> queryCompanies();
}

File: app/src/main/java/com/danilomendes/figurines/data/remote/api/CompanyService.kt
package com.danilomendes.figurines.data.remote.api

import com.danilomendes.figurines.data.entity.Company
import io.reactivex.Single
import retrofit2.http.GET

/**
 * Created by danilo on 07-12-2017.
 */
interface CompanyService {

    @GET("companies.json")
    fun queryCompanies(): Single<List<Company>>
}